Documentation/off-fault-receivers.rst

```diff
@@ -35,6 +35,8 @@ the form:
 The receivers files contain the time-histories of the stress tensor (6 variables) and the particle velocities (3).
 Currently, there is no way to write only a subset of these variables.
 
+Receiver time-histories are normally written at the end of the simulation, but this can be changed by setting :code:`ReceiverOutputInterval = 10.0`. Here the output is written every ReceiverOutputInterval seconds.
```

Comment:
What about:
"the variable :code:`ReceiverOutputInterval` (in the section :code:`Output` of the main parameter file) controls the frequency of flushing receiver time-histories. If not specified, they are written at the end of the simulation."